notebooks/phish_detection/Phishing_Detection_using_Bert_CLX.ipynb

```diff
@@ -73,10 +73,32 @@
      "name": "stderr",
```

Comment:
<p>Can we say a little more about the CLAIR dataset and provide a link to it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>